actuary.go

```diff
@@ -64,16 +64,24 @@ func main() {
 	}
 
 	actions := actuary.GetAuditDefinitions()
+
+	//log.Printf("XXXXXXXX %s", []actuary.Result)
+
 	//loop through the audits
 	for category := range tomlProfile.Audit {
-		log.Printf("Running Audit: %s", tomlProfile.Audit[category].Name)
+		if *output==""{
```

Comment:
Let's find a way to remove the `if *output==""{`'s. 

One option is controlling the place where an `Fprintln ` is printing out to (either `STDOUT` or some buffer that aggregates all the output) in the beginning, and we use `log.Fprintln` throughout actuary.